code/modules/food_and_drinks/machinery/microwave.dm

```diff
@@ -669,10 +671,34 @@
 			if(MICROWAVE_PRE)
 				pre_success(cooker)
 		return
+
+	if(cycles == 1) //Only needs to try to shock mobs once, towards the end of the loop
+		var/successful_shock
+		var/list/microwave_contents = list()
+		microwave_contents += src.get_all_contents() //Mobs are often hid inside of mob holders, which could be fried and made into a burger...
```

Comment:
```suggestion
		microwave_contents += get_all_contents() //Mobs are often hid inside of mob holders, which could be fried and made into a burger...
```

Reply:
If they aren't otherwise incapacitated they can just resist or walk out of the mob holder, removing them from the microwave, yeah.
They don't linger in the ingredients list when that happens